README.md

```diff
@@ -91,6 +91,49 @@ else
 }
 ``` 
 
+### Available User profile attributes through getters
+We have exposed user profile attributes through getters. You will find in the example below the getters available to you and how to use them:
```

Comment:
Not really, it's not specific to a user as it's for any user. But if you want to add it there I will add it.